.ci/travis/travis.sh

```diff
@@ -219,6 +219,10 @@ pr-description)
   .ci/travis/xtr_pr-description.sh
   ;;
 
+no-old-refs)
+  .ci/travis/no_old_refs.sh
```

Comment:
please move shell out of travis, it should not depend on certain CI, it should be (ideally) be agnostic to CI and be able to execute in any CI.
`/travis/` It was my bad design a long time ago, I did not thought at the beggining that Travis will be not the only CI we have.
We try to keep now all commands out of travis.sh, wercker.sh, ..... .

If possible it should be in separate shell or in tool/goal related scripts like pitest.sh, validation.sh, ..... 
This will let us more easily rebalance work between CIs when some CI had bad time or about to be discontinued.

Reply:
done, moved to .ci folder